run_action.py

```diff
@@ -538,6 +669,14 @@ def main():
         required=True,
         help="Number of suggestions per comment",
     )
+    parser.add_argument(
+        "--auto-resolve-conversations",
+        type=str,
+        required=False,
+        # this matches the YAML schema for the input
+        default="false",
```

Comment:
Considering that all other arguments are required for the Python script, even if they are not required by the YAML and have a default value there, what about the following in the name of consistency:

```suggestion
        required=True,
        help="If 'true', then close any discussions opened by the Action",
```